auth-request.lua

```diff
@@ -1,3 +1,6 @@
+-- From: https://raw.githubusercontent.com/TimWolla/haproxy-auth-request/c3c9349166fb4aa9a9b3964267f3eaa03117c3a3/auth-request.lua
+-- Changed to fit HAProxy Ingress needs - https://github.com/TimWolla/haproxy-auth-request/issues/35
+
```

Comment:
This should not be included here.

Reply:
arg, updating in a few minutes. came from my component. btw I don't need the second line anymore o/